src/Shell/AclShell.php

```diff
@@ -155,7 +155,7 @@ public function delete()
 
         $identifier = $this->parseIdentifier($this->args[1]);
         $nodeId = $this->_getNodeId($class, $identifier);
-        $entity = $this->Acl->{$class}->newEntity(['id' => $nodeId]);
+        $entity = $this->Acl->{$class}->get($nodeId);
```

Comment:
I see that the marshaller options are never passed onto the entity (I thought it was). I assume that this is for a reason, so `$entity->isNew(false)` is probably the best way to do it.